app/models/claim.rb

```diff
@@ -262,6 +263,10 @@ def school
     eligibility&.current_school
   end
 
+  def amendable?
+    submitted? && !decision&.rejected? && !passed_scheduled_payment_date?
```

Comment:
I think this is actually 
```suggestion
    submitted? && !decision&.rejected? && !payment.present?
```
since once we have imported the payment confirmation file there's nothing we can do, even if we import the file on a Tuesday and it's not scheduled to pay until the Friday

Reply:
Makes sense. Changed it, and also added an explicit check that the claim hasn't had its personal data removed.